src/types.js

```diff
@@ -21,3 +21,14 @@ export type _IdentifierType = {
 export type _Language = {
 
 };
+
+// Relationship Type Enum values
+export const RelationshipTypes = Object.freeze({
+	EditionContainsWork: 10,
+	NewEditionInEditionGroup: 3,
+	PublisherPublishedNewEdition: 4,
+	WrittenByAuthor: 8
```

Comment:
It would be good to implement each relationship in the relationship_type table.

For naming each one, I suggest using the `label` column from that same table.

Reply:
Hi @MonkeyDo , thanks a lot for the code review. I was eagerly waiting for it :) I've made the additional change you've requested in the commit below. Please let me know if any further changes need to be made.

Reply:
This is not what I meant, rather the other way around :)
Sorry I wasn't clear!

Do not change the relationship_type table at all, but instead rename the RelationshipTypes enum members to match the existing content of the relationship_type table:
Here's the entire relationship_type table. You'll need to create a property for each:

1	Worked On
2	Illustrator
3	Edition
4	Publisher
5	Editor
6	Inspiration
7	Parody
8	Author
9	Translator
10	Contains
11	    Marriage
12	Involved With
13	Member of Group
14	Parent
15	Sibling
16	Subgroup
17	Founder
18	Pen Name
19	Collaboration
20	Founded
21	Employee
22	Proofreader
23	Compiler
24	Designer
25	Typsetter
26	Photographer
27	Blurb
28	Art Director
29	Illustrator
30	Inker
31	Story
32	Letterer
33	Artist
34	Revisor
35	Reconstructed
36	Dedication
37	Previous Attribution
38	Commission
39	Licensor
40	Owner
41	Renamed
42	Split
43	Split
44	Copyright
45	Licensor
46	Licensee
47	Manufactured
48	Promoted
49	Distributed
50	Commission
51	Licensor
52	Revision
53	Quotation
54	Part
55	Derivative
56	Translation